tenable/io/v3/mssp/logos/api.py

```diff
@@ -0,0 +1,229 @@
+'''
+Logos
+=====
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`Logos <io-mssp-logos>` API endpoints.
+
+Methods available on ``tio.v3.mssp.logos``:
+
+.. rst-class:: hide-signature
+.. autoclass:: LogosAPI
+    :members:
+'''
+from io import BytesIO
+from typing import BinaryIO, Dict, List, Optional, Union
+from uuid import UUID
+
+from tenable.io.v3.base.endpoints.explore import ExploreBaseEndpoint
+from tenable.io.v3.mssp.logos.schema import LogoSchema
+
+
+class LogosAPI(ExploreBaseEndpoint):
+    '''
+    This class contains methods related to Logos API
+    '''
+    _path = 'api/v3/mssp/logos'
+    _conv_json = True
+    _schema = LogoSchema()
+
+    def add(self, logo: Union[BytesIO, BinaryIO], name: str) -> UUID:
+        '''
+        Adds a new logo
```

Comment:
Please update the doc string to contain more information. For example:
Adds a logo to the Tenable.io MSSP Portal
Ref: https://developer.tenable.com/reference/io-mssp-logos-create